editor/code_editor.cpp

```diff
@@ -657,9 +657,8 @@ FindReplaceBar::FindReplaceBar() {
 // be handled too late if they weren't handled here.
 void CodeTextEditor::_input(const Ref<InputEvent> &event) {
 	const Ref<InputEventKey> key_event = event;
-	if (!key_event.is_valid() || !key_event->is_pressed()) {
+	if (!key_event.is_valid() || !key_event->is_pressed() || !text_editor->has_focus())
```

Comment:
Please readd the braces, they are now enforced in our code style.

Reply:
Done! 
I wonder why the clang hook didn't catch that. 🤔